lightbulb/commands/base.py

```diff
@@ -595,6 +621,16 @@ def app_command_bypass_author_permission_checks(self) -> bool:
         """Whether invocations of this command as an application command will bypass author permission checks."""
         return self._initialiser.app_command_bypass_author_permission_checks
 
+    @property
+    def name_localizations(self) -> t.Mapping[t.Union[hikari.Locale, str], str]:
+        """A set of name localizations for this command"""
+        return self._initialiser.name_localizations
+
+    @property
+    def description_localizations(self) -> t.Mapping[t.Union[hikari.Locale, str], str]:
+        """A set of description localizations for this command"""
```

Comment:
Same as above

Reply:
Fixed in 05b489b88d4a2d8dc10a91c1a74c531fe867aacc.